doc/FORTANIX-SGX-ABI.md

```diff
@@ -2,6 +2,21 @@
 
 This document describes the ABI of SGX enclaves built using `libenclave`.
 
+## Enclave runner compatibility
+
+a. rust-lang toolchain version: https://github.com/rust-lang/rust/blob/master/src/libstd/sys/sgx/abi/entry.S
+```
+2:  .long 0                    /* desc - toolchain version number, 32-bit LE */
+```
+
+b. fortanix-sgx-tools version: https://github.com/fortanix/rust-sgx/blob/master/fortanix-sgx-tools/Cargo.toml
+
+| rust-lang toolchain version | fortanix-sgx-tools version | 
+| ---------------------------:| --------------------------:|
+|             1               |          0.4.0             |
+|             0               |          0.3.1             |
```

Comment:
Can you please add a reference to the Rust std version (by git commit) as well (see below for ABI compat)

Reply:
that is not delivered yet, rust-lang comes after rust-sgx, once it is merged I will add commit as well..

Reply:
Ok but you can add the column and fill it in for version 0. You can say TBD for version 1